roxctl/central/generate/validate.go

```diff
@@ -25,7 +25,17 @@ func validateHostPath(hostpath *renderer.HostPathPersistence) error {
 	if hostpath == nil {
 		return nil
 	}
-	if (hostpath.NodeSelectorKey == "") != (hostpath.NodeSelectorValue == "") {
+	if err := validateHostPathInstance(hostpath.Central); err != nil {
+		return err
+	}
+	return validateHostPathInstance(hostpath.DB)
```

Comment:
Could you add a unit tests for this change?

Reply:
done